packages/ledger-amino/src/ledgersigner.ts

```diff
@@ -37,6 +38,10 @@ export class LedgerSigner implements OfflineAminoSigner {
     return this.accounts;
   }
 
+  public async showAddress(path: HdPath): Promise<AddressAndPublicKeyResponse> {
```

Comment:
This should get its own return type that we can make public. `AddressAndPublicKeyResponse` depends on an external dependency, uses sname_case fields and is not exported from @cosmjs/ledger-amino.

Ideally we have this type on @cosmjs/amino-ledger:

```ts
export interface AddressAndPubkey {
    readonly address: string;
    readonly pubkey: Secp256k1Pubkey;
}
```

Reply:
@webmaster128 should this be done by declaring a new module for AddressAndPubkey or changing the existing interface?

Reply:
The existing interface cannot be changed because it is the format we get from `cosmos-ledger-js`, which is an external dependency.

You can just create the new interface as mentioned above in the same file.

Reply:
@webmaster128 ive added the interface to the signer itself, please let me know if this is the approach?